resources/static-folder/Cargo.toml

```diff
@@ -9,4 +10,4 @@ keywords = ["shuttle-service", "static-folder"]
 [dependencies]
 async-trait = "0.1.56"
 serde = { version = "1.0.148", features = ["derive"] }
-shuttle-service = { path = "../../service", version = "0.27.0" }
+shuttle-service = { path = "../../service", version = ">=0.27.0, <0.31.0" }
```

Comment:
What is this change for?

Reply:
This way, the final release 0.28 of static folder will stay compatible (but with deprecation compile warning) until 0.32 OR resource breaking change, whichever happen first.